xml/art_sle_ha_install_quick.xml

```diff
@@ -275,6 +275,16 @@
      <para>It opens the ports in the firewall that are needed for cluster communication.</para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term>Cluster Name</term>
+    <listitem>
+     <para>It defines a name for the cluster, by default
+        <systemitem>cluster<replaceable>NUMBER</replaceable></systemitem>. This
+      is optional and usefuly for &geo; clusters only. Usually, the cluster
+      name reflects its location and makes it easier to distinguish a site
```

Comment:
reflects its location -> reflects the location
('its' is ambiguous here because it could also refer to the 'name' which was the last-mentioned object in the phrase before)